subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -133,42 +135,60 @@
 #define CODED_PHY_CRC_SIZE               24
 #define CODED_PHY_TERM2_SIZE             3
 
-#define FEC_BLOCK1_TIME_US               ((CODED_PHY_ACCESS_ADDRESS_TIME_US) + \
+#define FEC_BLOCK1_US                    ((CODED_PHY_ACCESS_ADDRESS_TIME_US) + \
 					  (CODED_PHY_CI_TIME_US) + \
 					  (CODED_PHY_TERM1_TIME_US))
-#define FEC_BLOCK2_TIME_US(octets, mic)  (((((PDU_HEADER_SIZE) + \
+
+/* cs = 0, S2 coding scheme, use PHY_FLAGS_S2
+ * cs = 1, S8 coding scheme, use PHY_FLAGS_S8
+ *
+ * Not using the term CI, Coding Indicator, where in the Spec its defined value
+ * of 0 is S8 encoding, 1 is S2 encoding.
+ */
+#define FEC_BLOCK2_US(octets, mic, cs)   (((((PDU_HEADER_SIZE) + \
+					     (octets) + \
+					     (mic)) << 3) + \
+					   (CODED_PHY_CRC_SIZE) + \
+					   (CODED_PHY_TERM2_SIZE)) << \
+					  (1 + (((cs) & 0x01) << 1)))
```

Comment:
Can I assume that this is a value defined by the core spec? 

Otherwise, what does the value of `1` and `0x01` in line 153 mean?

Reply:
In S2 coding scheme, every bit is encoded into 2 bits that transmit on-air. When it is S8, every bit is transmitted in encode 8 bits. the `1` and `0x01` is used to accordingly calculate the required multiplication factor of 2 or 8.

The change here is replacing bitshift of `3` with `(1 + (((cs) & 0x01) << 1))`